.github/workflows/integration-test.yml

```diff
@@ -94,16 +94,21 @@ jobs:
           # this point.
           ref: ${{ github.event.pull_request.head.sha }}
 
-      - name: Install package with dependencies
-        uses: ./.github/actions/install-pkg
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3.2.2
         with:
-          python-version: ${{ matrix.python-version }}
+          enable-cache: true
+
+      - name: Setup nox
+        uses: wntrblm/nox@2024.10.09
+        with:
+          python-versions: ${{ matrix.python-version }}
 
       - name: Run integration tests
         env:
           EARTHDATA_USERNAME: ${{ secrets.EDL_USERNAME }}
           EARTHDATA_PASSWORD: ${{ secrets.EDL_PASSWORD }}
-        run: ./scripts/integration-test.sh
+        run: nox -s integration-tests -- --cov=earthaccess --cov-report=term-missing --capture=no --tb=native --log-cli-level=INFO
```

Comment:
I went back and forth on this myself and decided that I'd prefer not to, as I think most of these options are primarily useful for PR review and so would generally only appear in the CI/CD pipelines. 

We could provide a default set of positional arguments in the nox file if no positional arguments were provided, but it would be hard to pick and choose, and I'd error on the side of user flexibility. 

I've got another PR coming after this that documents nox testing a bit more, so we could document that we use those options in our CI/CD pipelines and why. That's coming as a separate PR because it's another big refactor of our dev guide and will likely spark a bit of conversation, which would be better kept in a separate PR.

Reply:
Sounds reasonable to me!